README.md

```diff
@@ -32,17 +32,42 @@ We support a growing list of platforms across the desktop, mobile, and console s
 * Mobile/Tablet Devices
   * Android 6.0 and up (OpenGL)
   * iPhone/iPad 10.0 and up (OpenGL)
-* Consoles (for registered developers)
+* [Consoles (for registered developers)](https://docs.monogame.net/articles/console_access.html)
   * PlayStation 4
   * PlayStation 5
   * Xbox One (XDK only) (GDK coming soon)
   * Nintendo Switch
 
+## Getting started
```

Comment:
Maybe change this title to something else?  Or the first "Getting started" below to something else.  Feels weird having two "Getting started"s right on each other.

Maybe `##Documentation` instead?